templates/csharp/xplat/AvaloniaTest.NetCore/AvaloniaTest.NetCore.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <RootNamespace>AvaloniaTest.NetCore</RootNamespace>
+    <AvaloniaVersion>0.10.999-cibuild0016975-beta</AvaloniaVersion>

```

Comment:
i can approve this PR only after https://github.com/AvaloniaUI/Avalonia/pull/7029 is merged because those templates depend on this PR heavily

Reply:
merged updated.